curriculum/challenges/english/25-front-end-development/lecture-working-with-backgrounds-and-borders/672b98db3bcdd545ab3b3c73.md

```diff
@@ -10,6 +10,62 @@ dashedName: what-are-the-different-ways-you-can-add-borders-around-images
 
 Watch the lecture video and answer the questions below.
 
+# --transcript--
+
+What Are the Different Ways You Can Add Borders Around Images?
+
+In CSS, there are several ways to add borders around images, each offering different styling options and levels of control.
+
+Let's explore some of the most common and versatile methods.
+
+The most straightforward way to add a border to an image is by using the border property. This property is a shorthand that allows you to set the width, style, and color of the border all at once.
+
+```css
+img {
+  border: 2px solid red;
+}
+```
+
+This CSS rule adds a 2-pixel wide, solid red border around all `img` elements. You can adjust the `width`, style (such as `dashed`, `dotted`, or `double`), and color to suit your design needs.
+
+Here is what the result looks like in the browser:
+
+![](https://cdn.hashnode.com/res/hashnode/image/upload/v1727379387306/14ad0ecb-4bfc-452e-bb2b-913389282fa6.png align="center")
```

Comment:
we should remove this since we decided not to include images for scripts.
It would be better to describe it instead

Reply:
I forgot to remove that, sorry.